disco/voice/client.py

```diff
@@ -78,7 +90,78 @@ def send_frame(self, frame, sequence=None, timestamp=None):
 
     def run(self):
         while True:
-            self.conn.recvfrom(4096)
+            data, addr = self.conn.recvfrom(4096)
+
+            # Check the packet size
+            if len(data) < 13:
+                raise ValueError('packet is too small: {}'.format(data))
+
+            # Unpack header
```

Comment:
I would move all of the below (perhaps even the length check) into a function, `decode` or the likes that can encapsulate the logic a bit better.